cmd/generate-config/incus_doc.go

```diff
@@ -313,7 +335,9 @@ func writeDocFile(inputJSONPath, outputTxtPath string) error {
 						configContentValue := configContent.(map[string]any)[configEntryContentKey]
 						if configEntryContentKey == "longdesc" {
 							backticksCount = countMaxBackTicks(configContentValue.(string))
-							longDescContent = configContentValue.(string)
+							if c, ok := configContentValue.(string); ok {
+								longDescContent = c
+							}
```

Comment:
Line break needed after this line.

Reply:
64a89a410249a7f8f65aef84d3f391d4fc211949